Solutions/GoogleCloudPlatformDNS/Hunting Queries/GCPDNSRareDomains.yaml

```diff
@@ -14,7 +14,7 @@ relevantTechniques:
 query: |
   GCPCloudDNS
   | where TimeGenerated > ago(24h)
-  | summarize count() by Query
+  | summarize count() by Query, SrcIpAddr

```

Comment:
In this case also, remove the SrcIpAddr as this hunting query is only looking for DNS query alone.